src/components/board/Stack.vue

```diff
@@ -161,14 +162,36 @@ export default {
 			this.editing = false
 		},
 		clickAddCard() {
-			const newCard = {
-				title: this.newCardTitle,
-				stackId: this.stack.id,
-				boardId: this.stack.boardId,
+
+			let match = /\r|\n/.exec(this.newCardTitle);
+			if (match) {
+				
+				match.forEach(() => (line) => {
```

Comment:
This is why it is not working, there is a a callback returned by your forEach callback which means that it will never be executed.

```suggestion
				match.forEach((line) => {
```